### PR TITLE
Add common interfaces to webNavigation events.

### PR DESCRIPTION
**Pull request by NN---**
Rename WebRequestCallbackDetails -> CallbackDetails , it is already in chrome.webRequest module.
Fix code formatting.

### CODE REVIEW

**Comment by vvakame** [comment]
@matthewkimber @otiai10 could you review this PR?

**Comment by otiai10** [comment]
yes sir

File: chrome/chrome.d.ts [hunk 1]
```diff
@@ -2095,88 +2095,61 @@ declare module chrome.webNavigation {
...
-    interface HistoryStateUpdatedDetails {
-        processId: number;
-        tabId: number;
+    interface CallbackUpdatedDetails extends CallbackDetails {
         transitionType: string;
-        url: string;
-        timeStamp: number;
-        frameId: number;
-        transitionQualifiers: string[];
+        transitionQualifiers: string;
```
Comment (otiai10) [inline chrome/chrome.d.ts]: Is it right `transitionQualifiers` is `string`?

Comment (NN---) [inline chrome/chrome.d.ts]: Copy paste bug :(
Thanks.

**Comment by otiai10** [comment]
It looks good to me. @vvakame

**Comment by vvakame** [comment]
@NN--- thanks mate!
@otiai10 thanks!

**Comment by otiai10** [comment]
:+1:

**Pull request merged**